docs/_includes/common/tooltip.html

```diff
@@ -0,0 +1,147 @@
+<div id="tooltip">
+    <h2><span class="secondary-text">@ebay/skin/</span>tooltip</h2>
+    <p>A tooltip is a flyout that visually <em>points</em> to a specific section or element of the page.</p>
+
+    <h4 id="tooltip-default">Default Tooltip</h4>
+    <p>The default tooltip, is the classic desktop tooltip that activates on focus and hover of a clickable element (typically an icon button or link). The purpose of the classic tooltip is to inform the user about the clickable element.</p>
+    <p>The default tooltip must be programmatically associated with it's clickable element by using the <span class="highlight">aria-describedby</span> property and <span class="highlight">tooltip</span> role.</p>
+    <p>Toggle the <span class="highlight">aria-expanded</span> state of the clickable element to expand or collapse it's tooltip.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="tooltip">
+                <button accesskey="s" class="tooltip__host icon-btn" aria-describedby="tooltip-0" aria-expanded="false" aria-label="Settings">
+                    <svg class="svg-icon" focusable="false" height="16" width="16" aria-hidden="true">
+                        <use xlink:href="#icon-settings"></use>
+                    </svg>
+                </button>
+                <div class="tooltip__overlay" id="tooltip-0" role="tooltip" style="top: -60px; left: -6px;">
+                    <span class="tooltip__pointer tooltip__pointer--bottom-left"></span>
+                    <div class="tooltip__mask">
+                        <div class="tooltip__cell">
+                            <div class="tooltip__content">
+                                <p>Use Access Key 'S' to display settings.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </span>
+        </div>
+    </div>
+    {% highlight html %}
+<span class="tooltip">
+    <button accesskey="s" class="tooltip__host icon-btn" aria-describedby="tooltip-0" aria-expanded="false" aria-label="Settings">
+        <svg class="svg-icon" focusable="false" height="16" width="16" aria-hidden="true">
+            <use xlink:href="#icon-settings"></use>
+        </svg>
+    </button>
+    <div class="tooltip__overlay" id="tooltip-0" role="tooltip">
```

Comment:
So, are we assuming, per the example above with `style="..."`, that we're expecting the developer to write/create the JS to handle positioning? If so, we should be explicit in our documentation that's we've manually positioned these examples.

Like a big note saying "You'll need to write a little bit of javascript to get the positioning correct."

Reply:
Yes, that assumption is correct. I will add that note.

Reply:
Awesome. Thought as much, and adding that will help it be clearer for developers.